packages/block-library/src/navigation/edit/accessible-description.js

```diff
@@ -0,0 +1,10 @@
+export default function AccessibleDescription( { id, content } ) {
+	return (
+		<div
+			id={ id }
+			className="wp-block-navigation__description screen-reader-text"
+		>
+			{ content }
```

Comment:
We should just use `children` as that's built into React.

Then you can just nested the compoentns.